lib/crypt.c

```diff
@@ -110,6 +110,18 @@ get_hashfn (const char *setting)
   return 0;
 }
 
+static int
+check_badsalt_chars (const char *setting)
+{
+  size_t i = 0;
```

Comment:
Redundant initialization. I'd keep one inside `for` only.

Reply:
Fixed in rebased commit.